arm-network/2016-09-01/swagger/examples/RouteFilterCreate.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
examples should be provided in the **examples folder under the api-version folder**.
arm-network/2016-09-01/ ~~swagger/~~ examples/RouteFilterCreate.json